doc/api/https.md

```diff
@@ -60,46 +66,36 @@ https.createServer(options, (req, res) => {
   res.writeHead(200);
   res.end('hello world\n');
 }).listen(8000);
-```
 
-Or
 
-```js
-const https = require('https');
-const fs = require('fs');
-
-const options = {
+// Or use a .pfx file which contains both the private key *and* certificate
+const optionsPfx = {
   pfx: fs.readFileSync('server.pfx')
 };
 
-https.createServer(options, (req, res) => {
+https.createServer(optionsPfx, (req, res) => {
   res.writeHead(200);
   res.end('hello world\n');
-}).listen(8000);
+}).listen(8001);
 ```
 
-### server.close([callback])
-<!-- YAML
-added: v0.1.90
--->
-
-See [`http.close()`][] for details.
-
-### server.listen(handle[, callback])
-### server.listen(path[, callback])
-### server.listen(port[, host][, backlog][, callback])
-
-See [`http.listen()`][] for details.
-
-## https.get(options, callback)
+## https.get(options[, callback])
 <!-- YAML
 added: v0.3.6
 -->
 
-Like [`http.get()`][] but for HTTPS.
+* `options` {Object | String} Contains request and connection details. If `options`
+  is a string, it is automatically parsed with [`url.parse()`]. If `options` is
```

Comment:
To me it makes it a little bit clearer that node is taking on the responsibility of calling `url.parse()` in that case.